### ISSUE TITLE
Toolkit page: Make the Guides section cards' SVGs WCAG compliant

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2929]
### Overview
We need to edit the guide card SVGs on the Toolkit webpage so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
For this issue, we will be adding the alt text property to the guide card images on the Toolkit webpage, which requires the HTML to be refactored.
- [x] Open _includes/guide-card.html in VS Code or your IDE, then change this line of code
  ```
  {% include {{ guide.svg }} %}
  ```
  to
  ```
  <img src={{ guide.svg | absolute_url }} alt="">
  ```
- [x] Move the `2FA.svg` file from `_includes\svg` to the `assets\images\guides\2FA` directory
- [x] Move the `survey-reporting-image.svg` file from `_includes\svg` to the `assets\images\guides\survey-reporting` directory
- [x] Update the svg field for `_guide-pages/2FA.html` from
  ```
  svg: svg/2FA.svg
  ```
  to
  ```
  svg: /assets/images/guides/2FA/2FA.svg
  ```
- [x] Update the svg field for `_guide-pages/survey-reporting-dashboard-guide.html` from
  ```
  svg: svg/survey-reporting-image.svg
  ```
  to
  ```
  svg: /assets/images/guides/survey-reporting/survey-reporting-image.svg
  ```
- [x] Update the svg field from
  ```
  svg: svg/default.svg
  ```
  to
  ```
  svg: /assets/images/toolkit-default-card-image.png
  ```
  for the following files in the `_guide-pages` directory:
  - [x] creating-kanban-project.html
  - [x] figma-for-opensource-projects.html
  - [x] github-issues.html
  - [x] responsible-use-of-images-on-opensource-projects.html
  - [x] setting-up-1password-on-opensource-project.html
  - [x] showing-civictech-experience-on-linkedin.html
  - [x] updates-team-pages-on-hflasite.html
  - [x] using-kanban-project.html
- [ ] Make sure the space between the image and text is there like how it is on the current live Toolkit webpage for mobile, tablet, and desktop views.
      <details>
        <summary>Click here to see the spacing which I have indicated with an arrow when viewing the page in desktop view</summary>
        <img width="537" alt="" src="https://user-images.githubusercontent.com/31293603/183505424-559c8cb6-05da-4c8d-be06-ff43bb3d6a87.png">
      </details>
  - [ ] If the spacing is not there, add the spacing back in; in VS Code or your IDE, edit the SCSS file `_sass/components/_toolkit.scss` and, if needed, edit the HTML in _includes/guide-card.html, such as the SCSS classes applied to the HTML
    - [ ] Make sure to reference the Figma design for the Toolkit page to determine the exact spacing for mobile and desktop views 
- [ ] Using Docker, make sure changes look good on mobile, tablet, and desktop views using developer tools and that nothing breaks
  - [x] #3723
    
### Resources/Instructions
- [Toolkit webpage](https://www.hackforla.org/toolkit)
- [_includes/guide-card.html](https://github.com/hackforla/website/blob/gh-pages/_includes/guide-card.html)
- [_includes/svg/2FA.svg file](https://github.com/hackforla/website/blob/gh-pages/_includes/svg/2FA.svg)
- [_includes/svg/survey-reporting-image.svg file](https://github.com/hackforla/website/blob/gh-pages/_includes/svg/survey-reporting-image.svg)
- Files in the `_guide-pages` directory:
  - [2FA.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/2FA.html)
  - [survey-reporting-dashboard-guide.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/survey-reporting-dashboard-guide.html)
  - [creating-kanban-project.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/creating-kanban-project.html)
  - [figma-for-opensource-projects.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/figma-for-opensource-projects.html)
  - [github-issues.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/github-issues.html)
  - [responsible-use-of-images-on-opensource-projects.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/responsible-use-of-images-on-opensource-projects.html)
  - [setting-up-1password-on-opensource-project.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/setting-up-1password-on-opensource-project.html)
  - [showing-civictech-experience-on-linkedin.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/showing-civictech-experience-on-linkedin.html)
  - [updates-team-pages-on-hflasite.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/updates-team-pages-on-hflasite.html)
  - [using-kanban-project.html](https://github.com/hackforla/website/blob/gh-pages/_guide-pages/using-kanban-project.html)
- [_sass/components/_toolkit.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_toolkit.scss)
- [Figma designs for the current webpages](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A0): You will need to zoom in and find the Toolkit page's designs (in red box) and screenshots. 
- Related design issue: #2567

**Comment by JessicaLucindaCheng** [comment]
@JessicaLucindaCheng See if you can use this: https://drive.google.com/drive/folders/1Pd07M-1hocCBrISiQ3x136-FivC9NLh1


Edit: It won't make a difference because this is an svg converted from a png. Thus, they are the same size. Also, the spacing issue is caused by original line of code in Liquid template language being `{% include {{ guide.svg }} %}`, which added the space after the image and before the header/title for each card. For more information on Liquid Template Language and spaces, see this documentation on [Whitespace control for Liquid template language](https://shopify.github.io/liquid/basics/whitespace/).

**Comment by JessicaLucindaCheng** [comment]
I may need to redo this issue. Instead of putting the SVGs into a img HTML tag, the SVGs should be made WCAG compliant by editing them directly or use JS to do it.

Edit: Never mind. Bonnie said if possible, placing them in img tags is the best solution.

**Comment by michaelmagen** [comment]
Availability: 4 hours
ETA: 11/6/22

**Comment by michaelmagen** [comment]
1. Working through tasks 
2. Time, working on finishing the task soon.
3. 4 hours
4. 11/15/22

**Comment by michaelmagen** [comment]
Blocker: Having trouble with fixing the spacing between the image and text after applying the changes.
